### PR TITLE
ignore Line Numbers

### PR DESCRIPTION
**Pull request by JehleM**
Option for ignoring numbers at the beginning of the line added.
grafik

### CODE REVIEW

File: src/Compare.cpp [hunk 1]
```diff
@@ -1128,7 +1128,8 @@ void ComparedPair::setStatusInfo()
 					options.ignoreSpaces		? TEXT(" Ignore Spaces ,")		: TEXT(""),
 					options.ignoreEmptyLines	? TEXT(" Ignore Empty Lines ,")	: TEXT(""),
 					options.ignoreCase			? TEXT(" Ignore Case ,")		: TEXT(""),
-					options.detectMoves			? TEXT(" Detect Moves ,")		: TEXT(""));
+					options.detectMoves			? TEXT(" Detect Moves ,")		: TEXT(""),
+					options.ignoreLineNumbers   ? TEXT(" Ignor Line Numbers ,") : TEXT(""));
```
Comment (MisterZeus) [inline src/Compare.cpp:1132]: Is the "Ignor" typo meant to be there?

File: src/UserSettings.cpp [hunk 2]
```diff
@@ -94,15 +95,15 @@ void UserSettings::load()
 
 	CharPrecision			= ::GetPrivateProfileInt(mainSection, charPrecisionSetting,		0, iniFile) != 0;
 	DiffsBasedLineChanges	= ::GetPrivateProfileInt(mainSection, diffsBasedChangesSetting,	0, iniFile) != 0;
-	IgnoreSpaces			= ::GetPrivateProfileInt(mainSection, ignoreSpacesSetting,		0, iniFile) != 0;
+	IgnoreSpaces			= ::GetPrivateProfileInt(mainSection, ignoreSpacesSetting,		0, iniFile) != 0;	
 	IgnoreEmptyLines		= ::GetPrivateProfileInt(mainSection, ignoreEmptyLinesSetting,	0, iniFile) != 0;
```
Comment (MisterZeus) [inline src/UserSettings.cpp:99]: extra tab whitespace at end of line?

**Comment by pnedev** [comment]
Hi @JehleM ,

Thank you for this useful contribution.
Thanks @MisterZeus for reviewing it.

I appreciate sharing this changes but I'll not merge the PR now because I'd like to stop adding new features until I finally release the first plugin version.
I would also like to implement a whole new advanced ignore settings as I mention in this **[PR](https://github.com/pnedev/compare-plugin/pull/230)**.
Please do not close the PR if possible, when the time comes I might use parts of it. Some users might find it useful as well and integrate it on their own.

Thanks once again.

BR

**Comment by pnedev** [comment]
Hello @JehleM ,

I have added Advanced Ignore function to ComparePlus plugin that I'm about to release in a month. It gives the possibility to ignore regex-es (that can be used to ignore numbers in lines beginnings as well).
Thank you once again for the PR +1

BR

**Pull request closed without merging**